lib/KMultiSelect.vue

```diff
@@ -0,0 +1,1084 @@
+          :class="['combobox-input', $computedClass(inputStyles)]"
+          :style="{
+            width: '100%',
+            height: '40px',
+            padding: autocomplete ? '0 40px 0 40px' : '0 40px 0 12px',
+            border: 'none',
+            outline: 'none',
+            fontSize: '14px',
+            color: $themeTokens.text,
+          }"
+          :placeholder="placeholder"
+          :aria-label="comboboxAriaLabel"
+          :aria-expanded="isDropdownOpen.toString()"
+          :aria-controls="listboxId"
+          :aria-activedescendant="getActiveDescendant()"
+          :aria-describedby="ariaDescribedById"
+          :aria-autocomplete="autocomplete ? 'list' : 'none'"
+          :aria-required="required ? 'true' : 'false'"
+          :aria-invalid="invalid ? 'true' : 'false'"
+          :disabled="disabled"
+          :readonly="!autocomplete"
+          @input="handleInput"
+          @keydown="handleComboboxKeydown"
+          @focus="handleInputFocus"
+          @blur="handleInputBlur"
+          @click="handleInputClick"
+        >
+
+        <KIconButton
+          v-if="searchText && autocomplete"
+          size="small"
+          icon="clear"
+          class="clear-search-button"
+          :style="{
+            position: 'absolute',
+            right: '32px',
+            top: '50%',
+            transform: 'translateY(-50%)',
+          }"
+          :ariaLabel="clearSearchMessage"
+          @click="clearSearch"
+        />
+
+        <KIconButton
+          size="small"
+          :icon="isDropdownOpen ? 'chevronUp' : 'chevronDown'"
+          class="dropdown-toggle"
+          :style="{
+            position: 'absolute',
+            right: '8px',
+            top: '50%',
+            transform: 'translateY(-50%)',
+          }"
+          :ariaLabel="getToggleDropdownLabel()"
+          :aria-expanded="isDropdownOpen.toString()"
+          @click="toggleDropdown"
+          @mousedown.prevent
+        />
+      </div>
+
+      <!-- Dropdown List -->
+      <div
+        v-show="isDropdownOpen"
+        ref="dropdownContainer"
+        class="dropdown-container"
+        :style="{
+          position: 'absolute',
+          top: '100%',
+          left: '0',
+          right: '0',
+          zIndex: '1000',
+          backgroundColor: $themeTokens.surface,
+          border: `1px solid ${$themeTokens.fineLine}`,
+          borderTop: 'none',
+          borderRadius: '0 0 4px 4px',
+          maxHeight: '400px',
+          overflowY: 'auto',
+        }"
+        role="region"
+        :aria-live="autocomplete ? 'polite' : 'off'"
+        @mousedown.prevent
+      >
+        <ul
+          :id="listboxId"
+          class="dropdown-list"
+          role="listbox"
+          aria-multiselectable="true"
+          :style="{
+            margin: '0',
+            padding: '0',
+            listStyle: 'none',
+            outline: 'none',
+          }"
+          :aria-labelledby="ariaLabelledby"
```

Comment:
When I navigated to the input, I didn't hear the screen reader to announce the context for what I’m actually choosing - 'Pick some fruits'. I wonder if it's just caused by `index.vue` example not providing `aria-labelledby`?